src/common/core/utils.py

```diff
@@ -84,6 +89,23 @@ def get_version_info() -> VersionInfo:
     return version_json
 
 
+def get_version_number() -> str:
+    """Returns the version number of the current deployment"""
```

Comment:
nit: Same as [here](https://github.com/Flagsmith/flagsmith-common/pull/59/files#r2060242893):

```suggestion
    """Return the version number of the current deployment"""
```